mapping/sustainability.yaml

```
@@ -1153,26 +1147,23 @@
     module: Climate finance
     indicator: KPIs
     disclosure format: It could be quarterly, annually, biannually, etc. \[free text\]
-    mapping: |-
-        Project level:
-
-        For each KPI metric add the sentence "To be reported XXX" where XXX is the reporting frequency to the metrics `.description`.
+    mapping: 'Project level: Get the [`Metric`](../../reference/schema.md#metric) object in the [`.metrics`](project-schema.json,,metrics) array that represents the KPI and map to its [`.description`](project-schema.json,/definitions/Metric,description) using the format "Reporting period: <period>".'
```

```suggestion
    mapping: |
        Project level: 
        
        Get the [`Metric`](../../reference/schema.md#metric) object in the [`metrics`](project-schema.json,,metrics) array that represents the KPI and map to its [`.description`](project-schema.json,/definitions/Metric,description) using the format "Reporting period: {period}".
```

"Reporting period: <period>" is becoming "Reporting period: " in the rendered page so need to use something else around the word to ensure it shows up